cmd/jaeger/internal/extension/jaegerquery/server.go

```diff
@@ -60,14 +60,13 @@ func (s *server) Start(ctx context.Context, host component.Host) error {
 	if err != nil {
 		return fmt.Errorf("cannot find primary storage %s: %w", s.config.Storage.TracesPrimary, err)
 	}
+	f = storagemetrics.NewDecoratorFactory(f, baseFactory)
```

Comment:
@yurishkuro this is will take in a `baseFactory` which will have the `query` namespace and then the decorator will add the `query` namespace. Just wanted to confirm that this is what we want? Note that the `queryMetricsFactory` is still being passed in when creating the server.

Reply:
we don't want `query` namespace twice. Perhaps we don't want it at all - why is it query and not storage?

Reply:
yes, if we can do that without affecting v1

Reply:
@yurishkuro I changed the decorator factory to namespace with `storage` and left `queryMetricsFactory` as is. Let me know what you think.